### DISCUSSION TITLE
[공지사항] ※2021-01-02 과제 및 공지사항※

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#58] [00_공지사항]
여러분 안녕하세요.

멘토 유혁준입니다. 👁‍🗨 (참조 : @4Moyede )

아래 과제들은 1/7일 오후 6시까지 모두 완료하여주시길 바랍니다.

## 1/2일 과제

1. **Notion 개인 포트폴리오 (일정 관리 페이지 or 여러가지) 만들어서 멘토들 공유로 초대하기**
2. 주차별 리뷰에서 
    1. **issue 들을 통해 어떤 것을 배웠는지**
    2. ~과제 review 보고 수정할 부분 수정해서 올리기~
    3. ~다른 멘티들의 과제를 보고 서로 리뷰해주기~
3. [lecture/01/20200102-AC_class_3_homework.ipynb](https://github.com/4Moyede/HexatonClass-01/blob/main/lecture/01/20210102-AC_class_3_homework.ipynb) 모두 실행하기
4. github 자기 repository, ex)Hexaton_Study 만들고 자기소개서 README.md 작성한 후, 자신의 remote repository에 push하기 (절대! HexatonClass-01 repository에 push 하지 말기) [discussion에 markdown 정리해둔 부분 참고하기](https://github.com/4Moyede/HexatonClass-01/discussions/18)

궁금한 점 있으면 댓글 달아주세요!

다음주 1/7일 목요일 오후 6시까지 올려주시면 (git push -> pull request) 됩니다.

멘토 유혁준

**Comment by yoomambo** [comment]
@4Moyede @lsj0822 @NA-DK @dlrudgnsdlek @jiwoo4424 @JaebeomHan 

![image](https://user-images.githubusercontent.com/54018187/103624719-4447e200-4f7d-11eb-8747-a23138ae3c43.png)

여러분 제 주피터노트북 파일에서는 1번문제의 출력물이 이런식으로 잘 나오는데,
업로드 된 파일은 A가 밀려보이네요...

이렇게 수정해서 푸시면 됩니다!

**Reply by yoomambo** [reply]
@lsj0822 @NA-DK @dlrudgnsdlek @jiwoo4424 @JaebeomHan 

1번 문제 착오없길 바랍니다~~

**Reply by ghost** [reply]
네엡

**Reply by lsj0822** [reply]
확인했습니다ㅏ

**Reply by jiwoo4424** [reply]
네 확인했습니다
